runtime/src/impls.rs

```diff
@@ -23,7 +23,7 @@ where
 {
 	fn on_unbalanceds<B>(mut fees_then_tips: impl Iterator<Item = NegativeImbalance<R>>) {
 		if let Some(fees) = fees_then_tips.next() {
-			// for fees, 100% to treasury, 0% to author
+			// for fees, 50% to treasury, 50%
```

Comment:
Where is the other %50 percent going?

Reply:
it is dropped, although we can setup a "Burn account" but i havent read into it

Reply:
https://github.com/UniversalDot/universal-dot-node/pull/107/commits/c43aaf5a431b2faef5988a19e5de050d3094a13a